NumberBaseball/NumberBaseball/main.swift

```diff
@@ -6,5 +6,53 @@
 
 import Foundation
 
-print("Hello, World!")
+typealias resultType = (strike: Int, ball: Int)
+
+func randomNumbers() -> Array<Int> {
+    var computerNumbers: Array<Int> = []
+    
+    while computerNumbers.count < 3 {
+        if computerNumbers.count == 3 {
+            break
+        }
+        let randomNumber = Int.random(in:1...9)
+        if computerNumbers.contains(randomNumber) {
+            continue
+        }
+        computerNumbers.append(randomNumber)
+    }
+    return computerNumbers
+}
+
+func compareNumbers(user userNumbers: Array<Int>, to computerNumbers: Array<Int>) -> resultType {
+    let strikes = userNumbers.enumerated().filter {userNumbers[$0.offset] == computerNumbers[$0.offset]}.map {$0.element}
+    let balls = userNumbers.filter {(strikes.contains($0) == false) && (computerNumbers.contains($0))}
+    let strike = strikes.count
+    let ball = balls.count
+    
+    return (strike: strike, ball: ball)
+}
+
+func menuSelect() {
+    var computerNumbers: Array<Int>
```

Comment:
```suggestion
    let computerNumbers: Array<Int> = randomNumbers()
```
위와 같이 let을 사용할 수도 있습니다.  
let과 var의 차이점에 대해서 아시나요?